Simplenote/src/main/java/com/automattic/simplenote/TagsListFragment.java

```diff
@@ -34,7 +34,7 @@
 
 import java.util.List;
 
-public class TagsListFragment extends ListFragment implements AdapterView.OnItemClickListener, ActionMode.Callback, AbsListView.MultiChoiceModeListener, AdapterView.OnItemLongClickListener, Bucket.Listener<Tag>{
+public class TagsListFragment extends ListFragment implements AdapterView.OnItemClickListener, ActionMode.Callback, AbsListView.MultiChoiceModeListener, AdapterView.OnItemLongClickListener, Bucket.Listener<Tag> {
```

Comment:
I notice those extra spaces or lack thereof everywhere too.  This might be another thing to put into another pull request or commit in order to keep the functional and style changes distinct.